Framework/API/src/FileProperty.cpp

```diff
@@ -216,7 +216,7 @@ bool FileProperty::isOptional() const {
 std::string FileProperty::setValue(const std::string &propValue) {
   std::string strippedValue = propValue;
   if (autoTrim()) { // strip whitespaces only when requested (default)
-    strippedValue = Kernel::Strings::strip(propValue);
```

Comment:
I think you were actually right to fix this though? The local variable here cannot be used anywhere so the strip never happens?

Reply:
Apologies for the confusion, but I cannot see this change when looking at the files changed. This must be a part issue of the reverting of some commits not necessarily in the proper order. Now, the `FileProperty` should not have any meaningful changes except of the imports cleanup. Because of the mess with reverting, perhaps it would be best to squash the commits when merging.

Reply:
Yes I think you are right. You fixed the local variable shadowing but then its now been reverted. As it's a bug I was wondering if we should reinstate that?

Reply:
The shadowing was introduced by the other commit actually, so the fix referred to that rather than an issue in the main branch.

Reply:
Ah I see. Sorry for the confusion. I'm happy with this and I'll merge it now.